handbook/onboarding/leads.md

```diff
@@ -84,9 +82,17 @@ Team formation happens in [kickoff events](/handbook/strategy/recurring-processe
 1. Create a **public** Slack channel using the [naming scheme](/handbook/tools/slack.md#naming-slack-channels) (e.g. `#tm-TEAMNAME`), and add all the developers and designers in your team to the channel.
    1. Make sure everyone on your team has set up their Rocket profiles, as per the [onboarding](/handbook/onboarding/everyone.md#rocket-setup).
 2. Run the following command in Slack so that Rocket can set up your GitHub team:
+
+   ```
+   /rocket team create TEAM_NAME --lead SLACK_HANDLE --channel TEAM_CHANNEL
    ```
-   /rocket team create $TEAM --lead $YOU --channel $CHANNEL
+
+   For example, if your team name is `Facebook for Cats`, your Slack handle is `@Jane Doe`, and the team Slack channel you created in step 1 is called `#tm-facebook-for-cats`, then you should write:
+
    ```
+   /rocket team create "Facebook for Cats" --lead @Jane Doe --channel #tm-facebook-for-cats
```

Comment:
Does the @Jane Doe work if there's a space? Do we also need to wrap that in quotations

Reply:
Yes, it does. I used a Slack handle with a space on purpose to illustrate this. Slack ends up recognizing it as a tag object like so: 
![image](https://user-images.githubusercontent.com/18269656/135138160-012e1d3e-54ae-43c4-a1af-50e99f24f2f5.png)

Reply:
Added note for clarity:
![image](https://user-images.githubusercontent.com/18269656/135139624-559e7ef8-f66d-493a-b072-a9eac4d33112.png)